src/pacemaker/doc/index.php

```diff
@@ -49,14 +49,6 @@
   <section class="docset">
     <h3 class="docversion">General Concepts (somewhat outdated)</h3>
     <table class="publican-doc">
-      <tr>
-        <td>Ordering Explained</td>
-        <td>[<a class="doclink" href="https://wiki.clusterlabs.org/w/images/a/ae/Ordering_Explained_-_White.pdf">pdf</a>]</td>
-      </tr>
-      <tr>
-        <td>Colocation Explained</td>
-        <td>[<a class="doclink" href="https://wiki.clusterlabs.org/w/images/8/8a/Colocation_Explained_-_White.pdf">pdf</a>]</td>
-      </tr>
```

Comment:
These files were outdated and not worth copying to the new wiki. I saved them so I can update the current docs with anything useful from them.